### PR TITLE
[cci] Copy the .dsc file to the folder

### PR DESCRIPTION
**Pull request by martin-g**
Add  _amd64|_arm64 before the extension

### CODE REVIEW

File: .circleci/make-deb-packages.sh [hunk 1]
```diff
@@ -49,4 +49,16 @@ dpkg-buildpackage -us -uc -j16
...
-mv ../*.deb ../*.dsc packages/$PARAM_DIST/$PARAM_RELEASE/
+mv ../*.deb packages/$PARAM_DIST/$PARAM_RELEASE/
+
+ARCH=`uname -m`
+ARCH2=""
+if [ "$ARCH" = "x86_64" ]; then
+  ARCH2="amd64"
+else
+  ARCH2="arm64"
+fi
```
Comment (gquintard) [inline .circleci/make-deb-packages.sh:60]:
```suggestion
if [ "`uname -m`" = "x86_64" ]; then
  ARCH="amd64"
else
  ARCH="arm64"
fi
```

**Comment by gquintard** [comment]
looks goog, thanks for beating me to it. Would you have a workflow featuring that change?

File: .circleci/make-deb-packages.sh (hunk 1 above)
Comment (martin-g) [reply .circleci/make-deb-packages.sh]: Applied!
I didn't use GitHub UI to apply your change because it needed to change the last line as well.

**Comment by martin-g** [comment]
> Would you have a workflow featuring that change?

I've created a new branch for this. The build will start soon at https://app.circleci.com/pipelines/github/martin-g/varnish-cache/299/workflows/537ee4e7-c8c1-4723-9f95-894c59353130

**Comment by gquintard** [comment]
great work, it's passing, so I'll merge as soon as you have this squashed.

I missed one thing during the big PR review, the versioning is a bit sketchy: `varnish_20200528-weekly~stretch-1` should possibly become `varnish_20200528-weekly-1~stretch` to keep the `SOURCEVERSION-PACKAGEVERSION` scheme. But that's a story for another ticket.

**Comment by martin-g** [comment]
Squashed and force pushed!

**Pull request merged**